docs/DTOmodels.md

```diff
@@ -73,3 +74,8 @@ metricUuids: string[]
 description: string[]
 isDone: boolean[]
 doneDate: (timestamp|null)[]
+
+WaysCollectionDTO #collection
```

Comment:
why do we need another collecntion?

Reply:
It's a subcollection/ We safw all stringified data to subcollections